src/routes/index.svelte

```diff
@@ -27,16 +27,16 @@
 
 <svelte:head>
   <title>Home – Innopoints</title>
-
-  <!-- Styles for Home page -->
-  <link rel="stylesheet" href="css/home/header.css" />
-  <link rel="stylesheet" href="css/home/main.css" />
-  <link rel="stylesheet" href="css/home/tagline.css" />
-  <link rel="stylesheet" href="css/home/how-to.css" />
-  <link rel="stylesheet" href="css/home/contacts.css" />
-  <link rel="stylesheet" href="css/home/options.css" />
-  <link rel="stylesheet" href="css/home/store.css" />
-  <link rel="stylesheet" href="css/page-components/empty-state.css" />
+  <link rel="stylesheet" href="/css/bundles/home.min.css" />
+  <link rel="preload" as="style" href="/css/bundles/store.min.css" />
+  <link rel="preload" as="style" href="/css/bundles/projects.min.css" />
+  {#if account}
+    {#if account.is_admin}
+      <link rel="preload" as="style" href="/css/bundles/dashboard.min.css" />
+    {:else}
+      <link rel="preload" as="style" href="/css/bundles/profile.min.css" />
+    {/if}
+  {/if}
```

Comment:
Here you want to use prefetch instead of preload
Check out https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content (look at the **Other resource preloading mechanisms** section)